tests/test_cloudevent_functions.py

```diff
@@ -0,0 +1,120 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import json
+import pathlib
+
+import cloudevents.sdk
+import cloudevents.sdk.event.v1
+import cloudevents.sdk.event.v03
+import cloudevents.sdk.marshaller
+import pytest
+
+from functions_framework import LazyWSGIApp, create_app, exceptions
+
+TEST_FUNCTIONS_DIR = pathlib.Path(__file__).resolve().parent / "test_functions"
+
+# Python 3.5: ModuleNotFoundError does not exist
+try:
+    _ModuleNotFoundError = ModuleNotFoundError
+except:
+    _ModuleNotFoundError = ImportError
+
+
+@pytest.fixture
+def event_1_0():
+    event = (
+        cloudevents.sdk.event.v1.Event()
+        .SetContentType("application/json")
+        .SetData('{"name":"john"}')
+        .SetEventID("my-id")
+        .SetSource("from-galaxy-far-far-away")
+        .SetEventTime("tomorrow")
+        .SetEventType("cloudevent.greet.you")
+    )
+    return event
+
+
+@pytest.fixture
+def event_0_3():
```

Comment:
use the term `cloudevent` rather than `event` everywhere where we are talking about a CloudEvent to disambiguate these signature types.

Reply:
Done